docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #68.                    QS = core.sliding_dot_product(Q, S)
</pre><p>Avoiding <code>S</code>, this would become <code>QT = core.sliding_dot_product(Q, T[start : stop - m + 1])</code> which I think is perfectly clear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>